src/utils/parser.mjs

```diff
@@ -130,5 +130,5 @@ export const parseHeadingIntoMetadata = (heading, depth) => {
     }
   }
 
-  return { text: heading, type: 'module', name: heading, depth };
+  return { text: heading, name: heading, depth };
```

Comment:
I wonder if this leads to headings without any type :)

Would suggest running a diff of the json of `main` and this branch.

Reply:
It does, but IMO that's the correct behavior, as we can't be certain the type is `module`, it could be `misc`, therefore, the type is empty. 

I could also not do this change, and have it overwrite the `module` with `misc` when needed (I.E. `json-legacy`).